pkg/controller/signup_test.go

```diff
@@ -217,3 +224,11 @@ func (m *FakeSignupService) GetSignup(userID string) (*signup.Signup, error) {
 func (m *FakeSignupService) CreateUserSignup(ctx *gin.Context) (*crtapi.UserSignup, error) {
 	return m.MockCreateUserSignup(ctx)
 }
+
+type FakeVerificationService struct {
+	MockSendVerification func(ctx *gin.Context, signup *crtapi.UserSignup) error
```

Comment:
This function is not used anywhere, is it?

Reply:
You're correct, it's not actually needed.  I've removed it.